SkillMatch/urls.py

```diff
@@ -12,13 +11,6 @@
     path('modify/', views.Modify),
     path('SearchUser/', views.SearchUser),
```

Comment:
```suggestion
    path('searchuser/', views.SearchUser),
```
Alban aime pas les majuscules